docker/Dockerfile.debian

```
@@ -55,6 +55,7 @@ RUN make PREFIX=${DIST_DIR} cmds
 
 WORKDIR $DIST_DIR
 COPY packaging/debian ./debian
+COPY deployments/systemd/ .
```

nit: Let's copy these into subfolder instead:
```suggestion
COPY deployments/systemd/ ./systemd
```